.env.test

```diff
@@ -19,6 +19,6 @@ EMAIL_HOST=
 EMAIL_PORT=
 EMAIL_HOST_USER=
 EMAIL_HOST_PASSWORD=
-BUGZILLA_HOST=
-BUGZILLA_API_KEY=
+BUGZILLA_HOST=https://bugzilla-dev.allizom.org
```

Comment:
Test shouldn't actually make any external calls, so I don't think we need this here.